src/agent/src/config.rs

```diff
@@ -180,6 +185,7 @@ impl FromStr for AgentConfig {
         config_override!(agent_config_builder, agent_config, server_addr);
         config_override!(agent_config_builder, agent_config, unified_cgroup_hierarchy);
         config_override!(agent_config_builder, agent_config, tracing);
+        config_override!(agent_config_builder, agent_config, aa_kbc_params);
```

Comment:
Do we need support to configure this parameter separately? It seems that it can only be configured via config_file now.

Reply:
@wllenyj that is what this macro does, it sets the variable to what was set in the config file. See #2517 for the introduction of this.